Sources/MapboxNavigation/Array.swift

```diff
@@ -1,4 +1,7 @@
 import Foundation
+import MapboxDirections
+import CoreLocation
+import Turf
```

Comment:
I'm not sure that this file should know about all Array extensions we have in our project. I would have created Array+RouteStep.swift file, but if this change seems normal for everyone else, we can go with it.